specification/azurefleet/resource-manager/Microsoft.AzureFleet/preview/2024-05-01-preview/azurefleet.json

```diff
@@ -465,14 +465,14 @@
     }
   },
   "definitions": {
-    "Azure.ResourceManager.Legacy.ManagedServiceIdentityType": {
+    "Azure.ResourceManager.CommonTypes.ManagedServiceIdentityType": {
       "type": "string",
       "description": "Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed).",
       "enum": [
         "None",
         "SystemAssigned",
         "UserAssigned",
-        "SystemAssigned, UserAssigned"
+        "SystemAssigned,UserAssigned"
```

Comment:
Since we have spec change and I want to confirm if the service side here would be `"SystemAssigned,UserAssigned"` not `"SystemAssigned, UserAssigned"`.

Reply:
Yes. Currently we only support UserAssigned identities. When customers pass systemAssigned the request will be rejected with appropriate error message from ARM.